px-app-nav.es6.js

```diff
@@ -56,14 +56,21 @@
        * and `label` defined. If an `icon` or `children` property is defined
        * on a subitem, those properties will be ignored.
        *
+       * The following optional properties can be used for any-level items:
+       *
+       * - {Boolean} isExternal - A boolean value for if this item should trigger
```

Comment:
`isExternal` only makes sense if a `link` prop is set (and `link` does not work if `isExternal` is missing). This makes this prop redundant.

I think we should remove this prop and replace it with:

- `href`: a string that contains whatever URL this item should navigate to
- `window`: an optional prop that determines what window opens the link. If it's unset, navigation should occur on the current page (no new tab/window etc). If it's set, that value is passed in as the 2nd param to `window.open`.

Thoughts?

Reply:
I like that idea, but disagree about the unset window behavior. I think most use uses of an external link would expect a new tab to open.

I would say use the identical behavior the `window.open()` spec outlines 
https://developer.mozilla.org/en-US/docs/Web/API/Window/open

Which is that empty `windowName` defaults to new tab/window.
`windowName='_self'` navigates the current window.
And `windowName='whatever'` open a new tab/window on first click and then renavigates that named window while it's still open.

Reply:
Yep, I agree.

Reply:
Changes we discussed are committed as well as the comment fix. Tried them on my local project, as worked well.